app/src/main/java/com/simplecity/amp_library/ShuttleApplication.java

```diff
@@ -378,4 +382,12 @@ private void enableStrictMode() {
                 .penaltyFlashScreen()
                 .build());
     }
-}
+
+    public static ShuttleApplication get() {
```

Comment:
Doesn't seem like these new methods and variables are used currently? Also, `mApp` is never assigned, so this will throw an NPE.

Reply:
Before our discussion in Teams, I thought this was the part that was generating UserIDs. So I will have the code written the future commits for it. Also prior to our discussion earlier today, do you think we need to capture the APP_UIDs for Muser now?

Reply:
No, we don't need the APP_UIDs in MUSER - that exists in OBA to generate a random ID on first execution and is sent with server API requests to get an idea of the number of clients server-side. We don't have an equivalent in MUSER, so we don't need it.

The UUIDs we do need are:
1. Firebase UUID for the user
1. Event UUIDs (generated from the Android UUID method)